### ISSUE TITLE
model_convert

### ISSUE THREAD

**Issue by ljh9999** [passion3394/darknet2caffe_yolov3#2]
Sorry to bother you again, I would like to ask if you have used the pre-training weight of darknet53.conv.74 to generate weights files and then convert caffemodel? I had a problem converting my own model. How do you train a model that fits your data set?

**Comment by passion3394** [comment]
No, I use the whole *.weights file to convert caffemodel. About the train process, you could refer this GitHub to train your custom dataset. https://github.com/AlexeyAB/darknet
